web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,29 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      reorder_visible: function() {
+          var visible = ''
+          var $itemList = $('.item-list');
+          var $visibleReports = $('.visible-reports');
+          for (var i = 0; i < fixmystreet.markers.features.length; i++) {
+              var feature = fixmystreet.markers.features[i];
+              var $listItem = $itemList.find('#report-'+ feature.data.id);
+              if (feature.onScreen()) {
+                  visible += $listItem[0].outerHTML; // This is faster than using .append()
+                  $listItem.addClass('hidden');
```

Comment:
On an `/around` page, this hides all the reports, I'm afraid. (As there's no visible-reports div, so they all get hidden from the main div.)
I'm not sure this is appropriate behaviour for the around page, which is already doing "what's on the map" (plus a few outside to make sure something is shown), should it be restricted to the reports listing page only?